### DISCUSSION TITLE
Want to add media to a meeting

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#1280] [Q&A] [answered]
Hi everyone,

On the meetings, I am able to a button to "add media" but when I click on it nothing happens. I'd really like to be able to upload groups flyers for special events or for monthly speaker meetings right to their meetings. Is there any way to do this?

Any help would be appreciated.

We still operate in Legacy UI, but will be making the switch to TSML UI in the near future.

**Comment by joshreisner** [comment, marked as answer]
the plugin doesn't support adding flyers or other related materials to meeting listings. 

most AA intergroups use a calendar plugin to promote one-off events like that, example: https://aasanjose.org/events, which uses the https://theeventscalendar.com plugin

**Comment by OSAwebsite** [comment]
Thanks for the clarification. Maybe this is something the plugin could work towards as it would be a helpful feature and good for the user experience to not have to look in a separate place for events tied to that meeting. 

We already use the events calendar, but there is a discussion happening around the equity of events being seen in the calendar as weekly "events" can tend to dominate the calendar over special events that are one time only.
